bug: startapp --with-pages does not generate ws_urls.rs required by #[routes] macro

## Summary

`reinhardt-admin startapp myapp --with-pages` does not generate `ws_urls.rs`, but the `#[routes]` macro in `src/config/urls.rs` expects `crate::apps::{app}::ws_urls::ws_url_resolvers` to exist for every registered app.

## Error

```
error[E0433]: failed to resolve: could not find `ws_urls` in `posts`
 --> src/config/urls.rs:8:1
  |
8 | #[routes]
  | ^^^^^^^^^ could not find `ws_urls` in `posts`

error[E0433]: failed to resolve: could not find `ws_urls` in `users`
 --> src/config/urls.rs:8:1
  |
8 | #[routes]
  | ^^^^^^^^^ could not find `ws_urls` in `users`
```

## Steps to reproduce

```sh
reinhardt-admin startproject myproject --with-pages
cd myproject
reinhardt-admin startapp posts --with-pages
reinhardt-admin startapp users --with-pages
cargo build
```

## Expected behavior

`startapp --with-pages` should generate `src/apps/myapp/ws_urls.rs` (with at least an empty `ws_url_resolvers` module) and add `pub mod ws_urls;` to the app module file (`src/apps/myapp.rs`), so that the project compiles out of the box.

## Workaround

Manually create `src/apps/posts/ws_urls.rs` and `src/apps/users/ws_urls.rs` with the following content, and add `pub mod ws_urls;` to each app module file:

```rust
pub mod ws_url_resolvers {}
```

--- a/crates/reinhardt-commands/templates/app_pages_template/lib.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_template/lib.rs.tpl
@@ -9,6 +9,8 @@ pub mod admin;
 pub mod models;
 pub mod serializers;
 pub mod urls;
+pub mod views;
+pub mod ws_urls;
 
 // Client-side modules
 #[cfg(client)]

--- a/crates/reinhardt-commands/templates/app_pages_template/models.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_template/models.rs.tpl
@@ -8,7 +8,9 @@
 //! ```rust,ignore
 //! use reinhardt::macros::user;
 //! use reinhardt::Argon2Hasher;
+//! use serde::{Deserialize, Serialize};
 //!
+//! #[derive(Serialize, Deserialize)]
 //! #[user(hasher = Argon2Hasher, username_field = "email")]
 //! #[model(table_name = "users")]
 //! pub struct User {

--- a/crates/reinhardt-commands/templates/app_pages_template/views.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_template/views.rs.tpl
@@ -1,0 +1,24 @@
+//! Views module for {{ app_name }} app (Pages)
+//!
+//! Define ViewSets here. Each `pub mod` declaration corresponds to a file
+//! under the `views/` directory.
+//!
+//! For multi-file views that need re-exports for discovery, use:
+//! ```rust,ignore
+//! flatten_imports! {
+//!     pub mod example;
+//! }
+//! ```
+//!
+//! # Example ViewSet
+//!
+//! ```rust,ignore
+//! use reinhardt::prelude::*;
+//! use reinhardt::viewset;
+//!
+//! // Import your model here
+//! // use crate::models::{{ camel_case_app_name }};
+//!
+//! #[viewset]
+//! pub struct {{ camel_case_app_name }}ViewSet;
+//! ```

--- a/crates/reinhardt-commands/templates/app_pages_template/ws_urls.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_template/ws_urls.rs.tpl
@@ -1,0 +1,29 @@
+//! WebSocket URL configuration for {{ app_name }} app
+//!
+//! Defines WebSocket consumer routes for this application.
+//! The `#[url_patterns(mode = ws)]` macro generates the `ws_url_resolvers` module
+//! consumed by the project-level `#[routes]` macro.
+//!
+//! # Example
+//!
+//! To register a WebSocket consumer:
+//!
+//! ```rust,ignore
+//! use reinhardt::url_patterns;
+//! use crate::config::apps::InstalledApp;
+//!
+//! #[url_patterns(InstalledApp::{{ app_name }}, mode = ws)]
+//! pub fn ws_url_patterns() {
+//!     // Register consumers via .consumer("path/", handler)
+//! }
+//! ```
+
+use reinhardt::url_patterns;
+
+use crate::config::apps::InstalledApp;
+
+#[url_patterns(InstalledApp::{{ app_name }}, mode = ws)]
+pub fn ws_url_patterns() {
+    // Register WebSocket consumers here.
+    // Example: .consumer("chat/", chat_consumer)
+}

--- a/crates/reinhardt-commands/templates/app_pages_workspace_template/lib.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_workspace_template/lib.rs.tpl
@@ -9,6 +9,8 @@ pub mod admin;
 pub mod models;
 pub mod serializers;
 pub mod urls;
+pub mod views;
+pub mod ws_urls;
 
 // Client-side modules
 #[cfg(client)]

--- a/crates/reinhardt-commands/templates/app_pages_workspace_template/models.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_workspace_template/models.rs.tpl
@@ -8,7 +8,9 @@
 //! ```rust,ignore
 //! use reinhardt::macros::user;
 //! use reinhardt::Argon2Hasher;
+//! use serde::{Deserialize, Serialize};
 //!
+//! #[derive(Serialize, Deserialize)]
 //! #[user(hasher = Argon2Hasher, username_field = "email")]
 //! #[model(table_name = "users")]
 //! pub struct User {

--- a/crates/reinhardt-commands/templates/app_pages_workspace_template/views.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_workspace_template/views.rs.tpl
@@ -1,0 +1,24 @@
+//! Views module for {{ app_name }} app (Pages)
+//!
+//! Define ViewSets here. Each `pub mod` declaration corresponds to a file
+//! under the `views/` directory.
+//!
+//! For multi-file views that need re-exports for discovery, use:
+//! ```rust,ignore
+//! flatten_imports! {
+//!     pub mod example;
+//! }
+//! ```
+//!
+//! # Example ViewSet
+//!
+//! ```rust,ignore
+//! use reinhardt::prelude::*;
+//! use reinhardt::viewset;
+//!
+//! // Import your model here
+//! // use crate::models::{{ camel_case_app_name }};
+//!
+//! #[viewset]
+//! pub struct {{ camel_case_app_name }}ViewSet;
+//! ```

--- a/crates/reinhardt-commands/templates/app_pages_workspace_template/ws_urls.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_workspace_template/ws_urls.rs.tpl
@@ -1,0 +1,29 @@
+//! WebSocket URL configuration for {{ app_name }} app
+//!
+//! Defines WebSocket consumer routes for this application.
+//! The `#[url_patterns(mode = ws)]` macro generates the `ws_url_resolvers` module
+//! consumed by the project-level `#[routes]` macro.
+//!
+//! # Example
+//!
+//! To register a WebSocket consumer:
+//!
+//! ```rust,ignore
+//! use reinhardt::url_patterns;
+//! use crate::config::apps::InstalledApp;
+//!
+//! #[url_patterns(InstalledApp::{{ app_name }}, mode = ws)]
+//! pub fn ws_url_patterns() {
+//!     // Register consumers via .consumer("path/", handler)
+//! }
+//! ```
+
+use reinhardt::url_patterns;
+
+use crate::config::apps::InstalledApp;
+
+#[url_patterns(InstalledApp::{{ app_name }}, mode = ws)]
+pub fn ws_url_patterns() {
+    // Register WebSocket consumers here.
+    // Example: .consumer("chat/", chat_consumer)
+}

--- a/crates/reinhardt-commands/templates/app_restful_template/models.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_restful_template/models.rs.tpl
@@ -8,7 +8,9 @@
 //! ```rust,ignore
 //! use reinhardt::macros::user;
 //! use reinhardt::Argon2Hasher;
+//! use serde::{Deserialize, Serialize};
 //!
+//! #[derive(Serialize, Deserialize)]
 //! #[user(hasher = Argon2Hasher, username_field = "email")]
 //! #[model(table_name = "users")]
 //! pub struct User {

--- a/crates/reinhardt-commands/templates/app_restful_workspace_template/src/models.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_restful_workspace_template/src/models.rs.tpl
@@ -8,7 +8,9 @@
 //! ```rust,ignore
 //! use reinhardt::macros::user;
 //! use reinhardt::Argon2Hasher;
+//! use serde::{Deserialize, Serialize};
 //!
+//! #[derive(Serialize, Deserialize)]
 //! #[user(hasher = Argon2Hasher, username_field = "email")]
 //! #[model(table_name = "users")]
 //! pub struct User {

--- a/website/content/quickstart/tutorials/basis/3-views-and-urls.md
+++ b/website/content/quickstart/tutorials/basis/3-views-and-urls.md
@@ -260,13 +260,7 @@ impl From<crate::apps::polls::models::Choice> for ChoiceInfo {
 
 ## Implementing Server Functions
 
-Create `src/server_fn.rs`:
-
-```rust
-pub mod polls;
-```
-
-Create `src/server_fn/polls.rs`:
+Create `src/apps/polls/server/server_fn.rs`:
 
 ```rust
 use crate::shared::types::{ChoiceInfo, QuestionInfo, VoteRequest};

